modin/numpy/arr.py

```diff
@@ -1546,7 +1565,140 @@ def astype(self, dtype, order="K", casting="unsafe", subok=True, copy=True):
         return array(_query_compiler=result, _ndim=self._ndim)
 
     def __repr__(self):
```

Comment:
this method's body is super-lengthy, can we split it into logical pieces, like `__repr_1dim()`, `__repr_1xN()`, `__repr_Nx1()`, etc., so that all this `if shape == foo` is not getting in the way of understanding what's happening?

Reply:
Sure!